metals/src/main/scala/scala/meta/internal/tvp/MetalsTreeViewProvider.scala

```diff
@@ -75,6 +76,12 @@ class MetalsTreeViewProvider(
     languageClient.metalsTreeViewDidChange(
       TreeViewDidChangeParams(
         Array(
+          TreeViewNode(
```

Comment:
Sort of nitpicky, but I'm trying to debug what is going on with `coc-metals` with the branch, and one things I noticed is that this is _technically_ against the spec that we have published. In the `TreeViewNode` class, `viewId` and `lable` aren't marked as `@Nullable`, and in the [spec](https://scalameta.org/metals/docs/editors/tree-view-protocol.html#tree-view-node) it's the same thing. I'm getting an undefined that I'm trying to chase down, and I don't think it's coming from here, but this is the sort of thing that could cause it and confuse someone since there should be somewhat of a guarantee that there is a `label` here. The same for the other two `TreeViewNode`'s below this.

Reply:
Added TreeViewNode.empty that sets both the id and label.